test/extended/openstack/loadbalancers.go

```diff
@@ -87,83 +90,88 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
-			nodeList, err := clientSet.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
-			o.Expect(err).NotTo(o.HaveOccurred())
-			expectedNumberOfMembers := len(nodeList.Items)
-			o.Expect(waitUntilNmembersReady(loadBalancerClient, pool, expectedNumberOfMembers, "NO_MONITOR|ONLINE")).NotTo(o.HaveOccurred(),
-				"Error waiting for %d members with NO_MONITOR or ONLINE OperatingStatus", len(nodeList.Items))
-
-			g.By("accessing the service 100 times from outside and storing the name of the pods answering")
-			results := make(map[string]int)
-			for i := 0; i < 100; i++ {
-				// https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/README.md#netexec
-				podName, err := getPodNameThroughUdpLb(svcIp, fmt.Sprintf("%d", svcPort), "hostname")
-				if err != nil {
-					e2e.Logf("Error detected while accessing the LoadBalancer service on try %d: %q", i, err)
-				} else {
-					results[podName]++
+				pool, err := pools.Get(loadBalancerClient, lb.Pools[0].ID).Extract()
+				o.Expect(err).NotTo(o.HaveOccurred())
+				o.Expect(pool.Protocol).Should(o.Equal(string(protocolUnderTest)), "Unexpected protocol on Openstack LoadBalancer Pool: %q", pool.Name)
+				//Set as OFFLINE in vexxhost despite the lb is operative
+				//o.Expect(pool.OperatingStatus).Should(o.Equal("ONLINE"), "Unexpected Operating Status on Openstack Pool: %q", pool.Name)
+				lbMethod, err := getClusterLoadBalancerSetting("lb-method", cloudProviderConfig)
+				o.Expect(err).NotTo(o.HaveOccurred())
+				o.Expect(strings.ToLower(pool.LBMethod)).Should(o.Equal(lbMethod), "Unexpected LBMethod on Openstack Pool: %q", strings.ToLower(pool.LBMethod))
+				nodeList, err := clientSet.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+				o.Expect(err).NotTo(o.HaveOccurred())
+				expectedNumberOfMembers := len(nodeList.Items)
+				o.Expect(waitUntilNmembersReady(loadBalancerClient, pool, expectedNumberOfMembers, "NO_MONITOR|ONLINE")).NotTo(o.HaveOccurred(),
+					"Error waiting for %d members with NO_MONITOR or ONLINE OperatingStatus", len(nodeList.Items))
+
+				g.By("accessing the service 100 times from outside and storing the name of the pods answering")
+				results := make(map[string]int)
+				for i := 0; i < 100; i++ {
+					// https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/README.md#netexec
+					podName, err := getPodNameThroughLb(svcIp, fmt.Sprintf("%d", svcPort), protocolUnderTest, "hostname")
+					if err != nil {
+						e2e.Logf("Error detected while accessing the LoadBalancer service on try %d: %q", i, err)
+					} else {
+						results[podName]++
+					}
 				}
-			}
-			e2e.Logf("Pods accessed after 100 UDP requests:\n%v\n", results)
-			pods, err := oc.KubeClient().CoreV1().Pods(oc.Namespace()).List(ctx, metav1.ListOptions{})
-			o.Expect(err).NotTo(o.HaveOccurred())
+				e2e.Logf("Pods accessed after 100 UDP requests:\n%v\n", results)
```

Comment:
looks like the mention to UDP is a leftover here

Reply:
Thanks for noticing @MaysaMacedo 
I created this PR to address this: https://github.com/openshift/openstack-test/pull/112/

Reply:
hey that's a wonderful opportunity for turning `port` into a number!